tests/test_dcml_import.py

```diff
@@ -0,0 +1,17 @@
+import unittest
+from partitura import load_dcml
+from tests import TSV_PATH
+import os
+
+
+class ImportDCMLAnnotations(unittest.TestCase):
+    def test_tsv_import_from_dcml(self):
+        note_path = os.path.join(TSV_PATH, "test_notes.tsv")
+        measure_path = os.path.join(TSV_PATH, "test_measures.tsv")
+        harmony_path = os.path.join(TSV_PATH, "test_harmonies.tsv")
+        score = load_dcml(note_path, measure_path, harmony_path)
+        self.assertEqual(len(score.parts), 1)
+
```

Comment:
maybe another assertion that counts the number of notes and compares with the number of lines in the tsv? Will they match?